lib/chrome-driver.js

```diff
@@ -24,7 +23,7 @@ ChromeDriver.prototype.start = function () {
     cwd: process.cwd(),
     env: this.getEnvironment()
   }
-  this.process = ChildProcess.spawn(process.execPath, args, options)
+  this.process = ChildProcess.spawn(this.path, args, options)
```

Comment:
That doesn't work for me.  I'm setting it when I construct the Application, right?  I have:

```
const app = new Application({
    path: ATOM_PATH,
    env: {
      ATOM_DEV_RESOURCE_PATH: '/Users/jonaldislarry/.atom',
      ATOM_SHELL_INTERNAL_RUN_AS_NODE: '1',
    },
});
```